distribution/openrct2.d.ts

```diff
@@ -260,6 +260,13 @@ declare global {
          */
         formatString(fmt: string, ...args: any[]): string;
 
+        /**
+         * Gets a string for for a shop item.
+         * @param shopItem The shop item.
+         * @param stringType The kind of string.
+         */
+        getShopItemString(shopItem: number, stringType: ShopItemStringType): string;
```

Comment:
Is this the right place in the API to put this method? I guess that in the future there will be a lot more methods to convert between ids and strings. We should spend 5 minutes thinking about how this will look like and if we want to have all of them here.
Maybe we should introduce a new API entry point so that `Context` does not get too crowded.

Reply:
Maybe a `Localization` namespace could be used to convert various IDs to String IDs and from String IDs to localized strings?

Reply:
We are currently discussing a localisation API on Discord #plugin. It will include a function `getLocalisedString(stringId: number): string`. So, this method here could be reduced to returning the string id. I still think that we should not put it here though, since the context namespace will get too crowded over time.

Reply:
@duncanspumpkin can you weigh in? I know you approved, but seems like the last comment was after that.